backend/pyproject.toml

```diff
@@ -58,7 +58,7 @@ openai = "^1.11.1"
 langchain = "^0.1.5"
 llama-index = "^0.10.45"
 tenacity = "^8.4.1"
-transformers = "^4.30.1"
+transformers = "^4.30.2"
```

Comment:
Yes, I noticed that and commented about it https://github.com/Chainlit/chainlit/pull/1250#issuecomment-2302969101.

I now rebased this on #1238, to it is dependent on that PR being merged first. Otherwise the build would still fail due to the PEP 517 issue with transformers.